### DISCUSSION TITLE
Export Data to Custom Excel Format

### DISCUSSION THREAD

**Discussion by JohnMroz** [dbeaver/dbeaver#16938] [Q&A]
Hello everyone,

Is it possible to export data to a customized Excel spreadsheet?

Thank you!

**Comment by LonwoLonwo** [comment]
Theese are all available fro Excel export setting in data transfer for data export customization.

![2022-07-13 10_24_20-DBeaver Ultimate 22 1 2 - DimEmployee](https://user-images.githubusercontent.com/45152336/178731270-7f2cd6ed-5f26-4e7c-810f-cff52ee7aca4.png)
